README.rst

```diff
@@ -54,10 +54,12 @@ motivations for the project
 in `our documentation <https://libcst.readthedocs.io/en/latest/index.html>`__.
 Try it out with `notebook examples <https://mybinder.org/v2/gh/Instagram/LibCST/master?filepath=docs%2Fsource%2Ftutorial.ipynb>`__.
 
+Example expression:
```

Comment:
If you want to make this line

```
Example expression::
```

it will display the same (single colon) and you don't need the `::` on the next line.  Up to you.